ttnn/examples/usage/using_dropout.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import ttnn
```

Comment:
If this is not going to be tested, this should be removed.
If it is important to have it, make sure this code is run on CI

Reply:
do you want me to remove example of using_dropout?
It is nice to have it to quickly check in future.

Reply:
It does not hurt to have an example.   Can we also have a unit test for this functionality?